packages/config/config/devices/0x0312/z97.json

```diff
@@ -0,0 +1,95 @@
+// NIE Technology Co., Ltd. ZW97 / Eva Logik
+// Outdoor Z-Wave Plug-In Module (2-Channel)
+// TODO: ZW97 support Over temperature protection, When the temperature is greater than 105 degrees Celsius, the Heat Alarm signal is sent out Notification Type : Heat Alarm 0x04 . Event :Overheat detected 0x02
+{
+	"manufacturer": "NIE Technology Co., Ltd.",
+	"manufacturerId": "0x0312",
+	"label": "ZW97",
+	"description": "Outdoor Z-Wave Plug-In Module (2-Channel)",
+	"devices": [
+		{
+			"productType": "0xc000",
+			"productId": "0xc007",
+			"zwaveAllianceId": 3701
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Outlet 1 (Left) State Notification",
+			"description": "When the state of Outlet 1 (Left) is changed, On (0xFF) or Off (0x00). The ZW97 will automatically send out a related command, On (0xFF) or Off (0x00) to its associated group.",
```

Comment:
We don't generally use descriptions for lifelines and I don't beleive any of the UIs do anything with them. These also don't conform to the style guide.

Reply:
I don't believe Groups 2 and 3 are considered lifelines.  "basic set command" is what the zwave alliance calls it. The style guide says to include a description if it adds value.  https://products.z-wavealliance.org/products/3701/assoc?noFilename=True  The description is rather verbose, I'll admit it can be simplified.
I wanted to convey that outlet 1 and 2 state notifications are sent on the respective groups.  I believe the intention was to allow other zw97s that are joined to group 2 or 3 to synchronize their outlet state.

Reply:
"Lifeline" is not the correct term, but we sometimes use it for additional association groups that should be associated to the controller

Reply:
I don't believe groups 2 and 3 would be associated with the controller. Most likely just other zw97s.

Reply:
@blhoward2 @AlCalzone Should I change the descriptions on groups 2 and 3? 

I just want those association groups to be have a meaningful description. Groups 2 and 3 wouldn't (or at least can't see a valid reason why it would) be associated with the controller.  Therefore not a "lifeline"(?)

Reply:
If there's a better description, go for it. Btw, I'm sure @blhoward2 meant "association group", not "lifeline".

Reply:
I meant Lifeline as the name of Group 1 per the style guide.

Reply:
Yes. I meant groups.

Reply:
@blhoward2  I removed the group descriptions. I think that's the last recommended change.